templates/repo/graph.tmpl

```diff
@@ -50,7 +50,7 @@
 				</div>
 			</h2>
 			<div class="ui dividing"></div>
-			<div class="ui segment loading gt-hidden" id="loading-indicator"></div>
+			<div class="is-loading tw-py-[120px]" id="loading-indicator"></div>
```

Comment:
Have you tested this?

Reply:
Yes, works fine, it enlarged the loading element just fine.

Reply:
That said, we could also just use py-32 which is close enough at 128px.

Reply:
Really "tested'? That comment was just my guess.

Now I checkout the branch, the page is broken:

![image](https://github.com/go-gitea/gitea/assets/2114189/9bf6e358-69ca-4d7e-9ef5-f256b398f01e)

Reply:
Ah I may have forgot to re-add `gt-hidden` during testing so that it's initially hidden like before.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/29920/commits/f5e632caf0e4a415214345c796093032a6a5bc25 and actually tested as well :)